tox.ini

```diff
@@ -27,6 +27,7 @@ deps =
      six
      coverage
 passenv =
+        ES_HOME
```

Comment:
Should we also add ES_HOSTPORT?

Reply:
`ES_HOSTPORT` is use when testing over an existing and running elasticsearch (faster when running it several times in local)
`ES_HOME` is used by testing.elasticsearch to setup a new instance for the time of unittests.
So, keep `ES_HOME`